docs/developing_iree/sanitizers.md

```diff
@@ -0,0 +1,58 @@
+# Using Address/Memory/Thread sanitizers
+
+## Enabling the sanitizers
```

Comment:
Please add a bit of an introduction explaining what sanitizers are and how they are useful

you can link to these docs:
https://clang.llvm.org/docs/AddressSanitizer.html
https://clang.llvm.org/docs/MemorySanitizer.html
https://clang.llvm.org/docs/ThreadSanitizer.html

the warnings like
>Typical slowdown introduced by AddressSanitizer is 2x.

would also be good to mention (so people don't always enable the sanitizers and later complain about performance issues)

Reply:
done